.ci/Jenkinsfile

```
@@ -204,13 +209,18 @@ def withKubernetes(Closure body) {
 
 def prepareStack() {
   def stackArgs = '-v'
-  def manifest = readYaml(file: "manifest.yml")
 
-  if (manifest != null && manifest["conditions"] != null) {
-    def kibanaVersionCondition = manifest["conditions"]["kibana.version"]
-    if (kibanaVersionCondition != null) {
-      def version = findOldestSupportedVersion(versionCondition: kibanaVersionCondition)
-      stackArgs += " --version ${version}"
+  if ( env.STACK_VERSION != '' ) {
+    stackArgs += " --version ${env.STACK_VERSION}"
```

Maybe you can use the `isTimerTrigger()` 
```suggestion
  if ( isTimerTrigger() ) {
    stackArgs += " --version 7.x-SNAPSHOT"
  } else if ( env.STACK_VERSION != '' ) {
    stackArgs += " --version ${env.STACK_VERSION}"
  }
```

---

Would it be possible to have something more explicit? Imagine for example that at some moment we also want to have scheduled jobs for 7.x-SNAPSHOT and 8.0-SNAPSHOT.

---

> Would it be possible to have something more explicit?

I guess a file with the different combination might be a good candidate.

OTOH, we did something similar for the Beats project:
- https://github.com/elastic/beats/blob/8afb80e6fa06fc9fce5fff1378b4756de90e647a/.ci/schedule-weekly.groovy#L21-L25

For such it was required the below steps:

1) Adapt this pipeline with the input parameters.
2) Create the Groovy scheduler definition in `.ci/schedule-weekly.groovy` or `.ci/schedule-daily.groovy` that called this pipeline with the different parameters
3) Create JJBB definition in `.ci/jobs/`

What do you think?

---

@v1v ok, this could be an option. But as proposed by Marcin in https://github.com/elastic/integrations/pull/1585#discussion_r703467918, would it be possible to install the parameterized scheduler plugin so this can be done in a cleaner way?

---

Infra folks are the one who manage the instance, raising a PR in their repo could be the first approach and ask them to install the plugin manually.

In my experience, the less number of plugins the more stable. If a similar approach can be done then I'd suggest to try it firstly.

---

Ok, sounds good, I will add the new definition, thanks!

---

@v1v I have added the definitions, no idea if they are correct :D could you please review?

---

Actually I am thinking now that maybe the groovy definition is not needed, it should be possible to parameterize the build also from the JJBB definition and use the main groovy definition, right?

---

Just a side note as I'm rather a spectator here. If the complexity of introducing this change is too high, maybe it's worth to postpone it for later?